### PR TITLE
Fix comments in UsersController

### PR DESCRIPTION
**Pull request by szepeviktor**
Just guessing!

### CODE REVIEW

**Comment by iamgergo** [comment]
This is intentional:

```
public function index(Request $request, User $user)// : Responsable
```

File: src/Http/Controllers/UsersController.php [hunk 1]
```diff
@@ -29,14 +29,14 @@ public function __construct(User $user)
...
-/**
+    /**
      * Display a listing of the resource.
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Bazar\Contracts\Models\User  $user
      * @return \Illuminate\Contracts\Support\Responsable
      */
-    public function index(Request $request, User $user)// : Responsable
+    public function index(Request $request, User $user): Responsable
```
Comment (szepeviktor) [inline src/Http/Controllers/UsersController.php:39]: This may need an additional|type.

**Comment by szepeviktor** [comment]
Done.

Do I understand correctly that the response type is this?
`\Illuminate\Contracts\Support\Responsable|\Illuminate\Http\JsonResponse`

**Comment by szepeviktor** [comment]
@iamgergo Friendly ping.

**Comment by iamgergo** [comment]
> Do I understand correctly that the response type is this?
> `\Illuminate\Contracts\Support\Responsable|\Illuminate\Http\JsonResponse`

Yep, will fix this.

**Pull request closed without merging**